docs/howto-mac.md

```diff
@@ -0,0 +1,65 @@
+# RLLIB SUMO Docker environment for MacOS
+
+## Step 1: Build your docker image
+
+Basic command to use:
+
+`bash docker-cmd-mac.sh --build --uid=502 --gid=222`
+
+- The default tag name of this image is `tf-gpu-sumo-{today_date}`. You can also add option such as `--image-name={new_name}` to specify your own name;
+- The default path of the dockerfile to be used is `docker-image-mac`. You can adapt to your setting by using option `--image-folder`;
+- The default image building process does not use cache. You can enable it by adding the option `--cache`.
+- The user id and group id are suggested as 502 and 222 respectively. The range of valid values on MacOS needs to be further studied.
+
+## Step 2: Run a container of the built image
+
+`sudo bash docker-cmd-mac.sh --image-name={your_image_name} --run --no-gpu`
+
+> :warning: **Using SUDO for docker is not a good practice**: Due to a problem of user privilege between host and container, `sudo` has to be used here to make it work. We will address this security issue soon.
```

Comment:
> We will address this security issue soon.

Are you sure it's addressable in the first place?
Otherwise, we can keep warning and remove the last statement, or we can say  "We will address this security issue, eventually.".

Reply:
I revised the last sentence accordingly c5d02771ca073099f34e6b7d3654b3b8035e8a8c